python/lsst/sconsUtils/builders.py

```diff
@@ -1,4 +1,5 @@
-"""Extra builders and methods to be injected into the SConsEnvironment class.
+"""Extra builders and methods to be injected into the SConsEnvironment
```

Comment:
Does this signify a doc-line-length configuration issue?

Reply:
It seems that black really wants to move the triple quote but then doesn't realize it has made it longer than the docstring length because black doesn't know about docstring lengths (or at least didn't used to know -- I will check if black 25 does know).

Reply:
Can't find anything.

Reply:
I wonder if ruff is better behaved?

Reply:
I've moved most of my code over to using ruff+ruff-format. Ruff-format is like black.

Reply:
I'm planning to move to ruff format soon with all middleware packages.